apps/spruce/src/pages/projectSettings/HeaderButtons.tsx

```diff
@@ -32,7 +32,6 @@ const defaultToRepoDisabled: Set<WritableProjectSettingsType> = new Set([
   ProjectSettingsTabRoutes.Plugins,
   ProjectSettingsTabRoutes.Containers,
   ProjectSettingsTabRoutes.ViewsAndFilters,
-  ProjectSettingsTabRoutes.GithubAppSettings,
   ProjectSettingsTabRoutes.GithubPermissionGroups,
```

Comment:
what's the reason the button is not be enabled for the permission groups tab?

Reply:
I set up the behavior so that if they default the app settings it will default all the permission settings too. It's not allowing users to default just the permission groups so that it doesn't end up in a messy state